Bottom: 9131c40d7e084acf1f70bb2ba8ba5199871de514
Top:    656c8c797c18df2868dfeb8c87cb06e5e157b176
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-25 19:50:58 +0000

Turn back on some commented 3D generation code


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 2eb8ea5..0e3db3b 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -255,8 +255,6 @@ get_contour_edge_n_is_round (PLINE *contour, int n)
     }
 #endif
 
-  return false;
-
   while (n > 0) {
     edge = edge->next; /* The VNODE structure is circularly linked, so wrapping is OK */
     n--;
@@ -1540,7 +1538,7 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
     END_LOOP;
 #endif
 
-#if 0
+#if 1
     fprintf (stderr, "Accumulating pin + via pads\n");
     r_search (PCB->Data->pin_tree, &bounds, NULL, pv_copper_callback, &info);
     r_search (PCB->Data->via_tree, &bounds, NULL, pv_copper_callback, &info);
@@ -1599,8 +1597,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
                               copper_appearance,
                               NULL));
 
-
-    break;
   }
 
 
diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 52182c0..2bb9bfd 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -202,11 +202,13 @@ step_do_export (HID_Attr_Val * options)
         poly_DelContour (&curc);
       }
 
+#if 1
     mask_objects = object3d_from_soldermask_within_area (piece, TOP_SIDE);
     board_outline_list = g_list_concat (board_outline_list, mask_objects);
 
     mask_objects = object3d_from_soldermask_within_area (piece, BOTTOM_SIDE);
     board_outline_list = g_list_concat (board_outline_list, mask_objects);
+#endif
 
     copper_layer_objects = object3d_from_copper_layers_within_area (piece);
     board_outline_list = g_list_concat (board_outline_list, copper_layer_objects);
